File: Domain/SynthesisBatch.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Domain
{
    public class SynthesisBatch
    {
        public Guid Id { get; set; }
        public String BatchNumber { get; set; }
        public DateTime Date { get; set; }
        public String StartTime { get; set; }
        public String EndTime { get; set; }
        public String TargetryPerson { get; set; }
        public String SynthesisPerson { get; set; }
        public String QCPerson { get; set; }
        public String Releaser { get; set; }
        public String Cyclotron { get; set; }

    }
}

File: client-app/src/app/models/synthesisbatch.ts
export interface SynthesisBatch {
    id: string;
    batchNumber: string;
    date: Date | null;
    startTime: string;
    endTime: string;
    targetryPerson: string;
    synthesisPerson: string;
    qcPerson: string;
    releaser: string;
    cyclotron: string;
}